rocky/onboarding/templates/step_2b_choose_report_type.html

```diff
@@ -29,19 +29,22 @@ <h2>{% translate "Choose a report - Type" %}</h2>
                         necessary steps.
                     {% endblocktranslate %}
                 </p>
-                <div class="button-container">
-                    {% if organization %}
-                        <a href="{% url "step_setup_scan_ooi_info" organization.code %}"
-                           class="button">{% translate "DNS report" %}</a>
-                    {% endif %}
-                    <a href="" class="button disabled">Pen test</a>
-                    <a href="" class="button disabled">Mail report</a>
-                    <a href="" class="button disabled">DigiD</a>
-                    {% if organization %}
-                        <a href="{% url "complete_onboarding" organization.code %}"
-                           class="button ghost">{% translate "Skip onboarding" %}</a>
-                    {% endif %}
-                </div>
+                <form method="get"
+                      action="{% url "step_setup_scan_ooi_info" organization.code %}"
+                      class="inline">
+                    <div class="button-container">
+                        {% if organization %}
+                            <button name="report_type" value="dns-report" type="submit" class="button">{% translate "DNS report" %}</button>
+                        {% endif %}
+                        <a href="" class="button disabled">Pen test</a>
```

Comment:
Could we list the available reports instead of making them up? All but the DNS can still be disabled but this way we can use the same code in the onboarding as we will on the report flow.

Reply:
Yes I was thinking of this indeed. Pass all the Reports we want to show in onboarding through from views to template

Reply:
We need to do this in another PR, there is more complexity to it. Choosing a different report type, means that we need to populate a different form to create the desired OOI that applies to that report type. DNS report would need a Hostname fetched from URL, but other report types needs different forms. Also the forms are nested, you need to create this first to create the second and so on ....

Reply:
Allowing *just* the dns report i the onboarding is fine. But lets keep this PR small, and create a new issue for that.